src/main/public/js/analytics-custom-event-sender.js

```diff
@@ -30,13 +30,7 @@ $(function () {
       throw new Error('Label element is required')
     }
 
-    for (var content of labelElement.contents()) {
-      const text = $(content).text().trim()
-      if (text && content.class !== 'form-hint') {
-        return text
-      }
-    }
-    return undefined
+    return labelElement.contents()[0].textContent.trim()
```

Comment:
are we sure this will always complete successfully?

Reply:
nope, needs a bit more testing really

Reply:
it works for eligibility